src/index.scss

```diff
@@ -1,14 +1,14 @@
 body {
```

Comment:
I don't like that all of these changes are on the main `index.scss` file. They might affect all other skins. Can this be scoped to your skin only?